src/linter/xmlTemplate/Parser.ts

```diff
@@ -563,16 +563,28 @@ export default class Parser {
 					line: prop.start.line + 1, // Add one to align with IDEs
 					column: prop.start.column + 1,
 				};
-				if (this.#apiExtract.isProperty(symbolName, prop.name)) {
-					this.#bindingLinter.lintPropertyBinding(prop.value, this.#requireDeclarations, position);
-				} else if (this.#apiExtract.isAggregation(symbolName, prop.name)) {
+				if (this.#apiExtract.isAggregation(symbolName, prop.name)) {
 					this.#bindingLinter.lintAggregationBinding(prop.value, this.#requireDeclarations, position);
 				} else if (this.#apiExtract.isEvent(symbolName, prop.name)) {
+					// In XML templates, it's possible to have bindings in event handlers
+					// We need to parse and lint these as well
+					this.#bindingLinter.lintPropertyBinding(prop.value, this.#requireDeclarations, position);
+
 					EventHandlerResolver.parse(prop.value).forEach((eventHandler) => {
 						if (eventHandler.startsWith("cmd:")) {
 							// No global usage possible via command execution
 							return;
 						}
+						// Check for a valid function/identifier name
+						// Currently XML views support the following syntaxes that are covered with this pattern:
+						// - myFunction
+						// - .myFunction
+						// - my.namespace.myFunction
+						// - my.namespace.myFunction(arg1, ${i18n>key}, "test")
+						const validFunctionName = /^\.?[$_\p{ID_Start}][$\p{ID_Continue}]*(?:\.[$_\p{ID_Start}][$\p{ID_Continue}]*)*(\(.*\))?$/u;
```

Comment:
As we have a regular expression now to check whether there is a valid function name, why don't we also use it to extract that name instead of checking for it again below? This would reduce the code and ensures consistency.

Reply:
Done